pkg/interface/chat/src/js/components/skeleton.js

```diff
@@ -18,36 +18,60 @@ export class Skeleton extends Component {
     ? "dn-s"
     : "";
 
+    // mobile-specific navigation classes
+    let mobileNavClasses = classnames({
+      "dn": this.props.chatHideOnMobile,
+      "db dn-m dn-l dn-xl": !this.props.chatHideOnMobile,
+      "w-100 inter pt4 f8": !this.props.chatHideOnMobile
+    })
+
+    // popout switches out window chrome and borders
+    let popoutWindow = this.props.popout
+    ? ""
+    : "ph4-m ph4-l ph4-xl pb4-m pb4-l pb4-xl";
+
+    let popoutBorder = this.props.popout 
+    ? "" 
```

Comment:
ternary statements are fine, but should not use this style.

```let popoutWindow = this.props.popout ? "" : "asdfsdf";```

or

```
let popoutWindow = this.props.popout
  ? "" : "asdfsdfsadf";
```

or 

```
let popoutWindow = this.props.popout ? 
  (
    "asdfasdffsd" +
    "asfdkjlfsdaj"
  ) : "asdfsdfsadf";
```

any of these would be fine.
```